recipes/xapian-core/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "xapian-core" %}
```

Comment:
It seems like this would just be called xapian

Reply:
Hm, `xapian-core` is the upstream name for the core C++ package, "Xapian" would be the whole thing (with bindings and what else is included).

Reply:
Ahh then leaving this name is fine.